dp/generate_climos.py

```diff
@@ -394,3 +423,18 @@ def update_metadata_and_time_var(input_file, t_start, t_end, climo_filepath):
         climo_bnds_var[:] = date2num(climo_bounds, cf.time_var.units, cf.time_var.calendar)
 
     return climo_filepath
+
+
+def validate_operation(operation):
+    """
+    Given an operation assert that it is supported.  In order to be a supported
+    operation it must be in the cdo table of statistical values.
+    """
+    supported_operations = {
+        'mean',
+        'std'
+    }
+    if operation in supported_operations:
+        return True
+    else:
+        return False
```

Comment:
This could be simplified to 

```python
return operation in supported_operations
```
The next item overrides this, but it's worth pointing out that this is pretty verbose and unnecessary.

Reply:
That aside, you don't appear to do anything other than raise an exception when the test fails. So raise that exception inside this function and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) up your usage code.